### PR TITLE
DIV-5457:Add IssuedToBailiff and AwaitingBailiffService states for new page

### PR DESCRIPTION
**Pull request by jacmurphy**
# Description

```
GIVEN a case is in the state 'IssuedToBailiff'
AND the case is a petitioner case
WHEN the petitioner logs into PFE
THEN they are taken to a landing page with the copy shown below
AND if the respondent is male then they are referred to conditionally as 'husband'
AND if the respondent is female then they are referred to conditionally as 'wife'


GIVEN a case is in the state 'AwaitingBailiffService'
AND the case is a petitioner case
WHEN the petitioner logs into PFE
THEN they are taken to a landing page with the copy shown below
AND if the respondent is male then they are referred to conditionally as 'husband'
AND if the respondent is female then they are referred to conditionally as 'wife'
```

### CODE REVIEW

File: app/middleware/submissionMiddleware.test.js [hunk 1]
```diff
@@ -91,6 +93,18 @@ describe(modulePath, () => {
...
+      ['AwaitingBailiffService']
+    ])
+      .it('should redirect to `/issued-to-bailiff` if in is %s', caseState => {
+        req.session.caseId = TEST_CASE_ID;
+        req.session.state = caseState;
+        underTest.hasSubmitted.apply(ctx, [req, res, next]);
+        expect(res.redirect.calledOnce).to.eql(true);
+        expect(res.redirect.calledWith(SENT_TO_BAILIFF)).to.eql(true);
+      });
+
```
Comment (jacmurphy) [inline app/middleware/submissionMiddleware.test.js:107]: Both states will bring the user to the same page

File: test/end-to-end/data/issuedToBailiffSession.js [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    ft_awaiting_amend: true,
+    ft_represented_respondent_journey: true,
+    ft_welsh: true
+  },
+  expires: 99999999,
+  language: 'en',
+  refusalRejectionReason: [],
+  caseReference: 'RS34D82873',
+  state: 'IssuedToBailiff',
+  caseId: '1594218147345457',
```
Comment (jacmurphy) [inline test/end-to-end/data/issuedToBailiffSession.js:21]: Included the ticket number in the last 4 digits

File: app/middleware/submissionMiddleware.test.js [hunk 3]
```diff
@@ -14,6 +14,8 @@ const DONE_AND_SUBMITTED = '/done-and-submitted';
 const APPLICATION_MULTIPLE_REJECTED_CASES_PATH = '/contact-divorce-team';
 const AMENDMENT_EXPLANATORY_PAGE = '/amendment-explanatory-page';
 const CONTACT_DIVORCE_TEAM_PATH = '/contact-divorce-team';
+const SENT_TO_BAILIFF = '/issued-to-bailiff';
+
```
Comment (yarpo) [inline app/middleware/submissionMiddleware.test.js:18]: redundant empty line

File: app/steps/issued-to-bailiff/content.json [hunk 4]
```diff
@@ -0,0 +1,49 @@
+{
+  "lng": "en",
+  "resources": {
+    "en": {
```
Comment (yarpo) [inline app/steps/issued-to-bailiff/content.json:4]: what about Welsh?

File: app/steps/issued-to-bailiff/content.json [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+            "deemedAsServedGranted": "Deemed service order",
+            "dispenseWithServiceGranted": "Order to dispense",
+            "deemedServiceRefused": "Deemed service refusal",
+            "dispenseWithServiceRefused": "Dispensed service refusal",
+            "generalOrder": "General Order"
+          }
+        },
+        "errors": {}
+      }
+    }
```
Comment (yarpo) [inline app/steps/issued-to-bailiff/content.json:47]: you should have `"cy": {` here and add the same values again, like in:
div-petitioner-frontend/app/steps/application-submitted-awaiting-response/content.json

File: app/middleware/submissionMiddleware.test.js [hunk 6]
```diff
@@ -91,6 +93,18 @@ describe(modulePath, () => {
       expect(res.redirect.calledWith(CONTACT_DIVORCE_TEAM_PATH)).to.eql(true);
     });
 
+    forEach([
+      ['IssuedToBailiff'],
+      ['AwaitingBailiffService']
+    ])
+      .it('should redirect to `/issued-to-bailiff` if in is %s', caseState => {
```
Comment (madjava) [inline app/middleware/submissionMiddleware.test.js:100]: I guess you meant to say `'should redirect to `/issued-to-bailiff` if is %s'`

File: app/steps/issued-to-bailiff/content.json (hunk 5 above)
Comment (madjava) [reply app/steps/issued-to-bailiff/content.json:47]: Yup do add a CY section to be on the safe side

**Review by madjava — APPROVED** [review]

**Review by Dizrupto — APPROVED** [review]

File: test/end-to-end/paths/IssuedToBailiff.js [hunk 7]
```diff
@@ -0,0 +1,13 @@
+const content = require('app/steps/issued-to-bailiff/content.json').resources.en.translation.content;
+const whoResponds = content.yourResponds.replace('{{ divorceWho }}', 'wife');
+
+Feature('IssuedToBailiff - Sent to Bailiff path @functional').retry(2);
```
Comment (yarpo) [inline test/end-to-end/paths/IssuedToBailiff.js:4]: I'd move it to nightly.

is:
```
Feature('IssuedToBailiff - Sent to Bailiff path @functional').retry(2);
```

expected:
```
Feature('IssuedToBailiff - Sent to Bailiff path @nightly').retry(3);
```

I'd also increase retry to 3.

**Pull request merged**